bridge_adaptivity/module/models.py

```diff
@@ -42,7 +40,26 @@ def _get_engine_driver(engine):
     return driver
 
 
-ENGINES = _discover_engines()
+def _load_cls_from_applicable_module(module_path, mod_name, class_startswith=None, class_endswith=None):
+    """Load class from module."""
+    module = None
+    gp_module = importlib.import_module('{}{}'.format(module_path, mod_name))
```

Comment:
I would add period in the middle and create the path from the path to the module dir and module name:
```importlib.import_module('{}.{}'.format(module_path, mod_name))```

Reply:
it would  work for importing engines, but would not for policies.